### ISSUE TITLE
Video tutorial on using the debugger

### ISSUE THREAD

**Issue by tap** [Cycling74/min-devkit#99]

**Comment by Jonny0815** [comment]
Hello, im currently writing a node for my bachelor thesis using this and its working pretty fine, thanks!

Can you give a quick write-up on how to debug my code as the video tutorial isnt available yet?

**Comment by tap** [comment]
Hi.  This is what I do...

For the ALL BUILD target, right-click on it and choose "Properties":

<img width="355" alt="image" src="https://user-images.githubusercontent.com/109637/61492401-6fe0a200-a96e-11e9-9448-513e28663865.png">

Then choose "Debugging" and for the command choose "Browse" :

<img width="592" alt="image" src="https://user-images.githubusercontent.com/109637/61492504-ac140280-a96e-11e9-9332-a8759bc2dd00.png">

In the dialog that ensues, locate the copy of Max.exe you'd like to debug with. Then Click "okay" to close the properties box.

Now, to debug simply press this button:

<img width="163" alt="image" src="https://user-images.githubusercontent.com/109637/61492592-e4b3dc00-a96e-11e9-9606-f9bed06b3aeb.png">

Hope this helps!

**Comment by Jonny0815** [comment]
ou shoot, i completely forgot to mention that im using max 4 live ...

can i somehow debug my object while it is in a "max audio effect" inside ableton live?

thanks for all your help :)

**Comment by tap** [comment]
Yes. All you need to do is choose Live instead of Max when you browse for
the app to debug with.

Cheers!
  Tim
